modules/nfc-reader/src/NfcReader.types.ts

```diff
@@ -0,0 +1,32 @@
+export type NfcData = {
```

Comment:
Where is this type used? either this one or the type in passport-data.tsx might be redundant